db/test/test_dataset.py

```diff
@@ -119,6 +123,22 @@ def test_get_by_user_id(self):
         datasets = dataset.get_by_user_id(self.test_user_id, public_only=False)
         self.assertEqual(len(datasets), 2)
 
+    def test_add_recordings(self):
+
+        id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        ds = dataset.get(id)
+
+        dataset.add_recordings(self.test_data_recordings)
```

Comment:
The reason for adding tests is to perform some action and then to verify that this action has been successfully applied.
In this case you are calling a method but then not doing anything with it.

For me, a test for this method would do something like this:

1. Create a test dataset
2. Call the `add_recordings` method to add some new recording ids to a class
3. Get the dataset from the database again and check that it contains both the recordings which were originally in it plus the new ones.
4. An optional test, related to an earlier comment: Check that if there is a separate dataset with the same named class, that it was not affected.

Two other details in this method:
 * You call `ds = dataset.get(id)` but do nothing with this data. Make sure you don't have unused code in methods
 * We use dictionaries in `self` for data which is shared between many tests. In this case, `self.test_data` is used in many places and so we initialise it in the `setUp` method. In the case of the `test_data_recordings` dictionary, this is only used in this test, so you can define it in the method.